src/main/java/doke/Storage.java

```diff
@@ -0,0 +1,58 @@
+package doke;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * a class to represents the storage.
+ */
+public class Storage {
+    protected static final File DOKE_FILE = new File(Doke.DOKE_FILE_PATH);
```

Comment:
All capitalized name suggests that it is a constant instead of a variable. Good.